app/models/gemini_connection.rb

```diff
@@ -3,20 +3,21 @@
 class GeminiConnection < ApplicationRecord
   include WalletProviderProperties
 
-  SUPPORTED_CURRENCIES = ["BAT", "USD", "BTC", "ETH"].freeze
   JAPAN = 'JP'
 
   has_paper_trail
 
   belongs_to :publisher
+  has_many :gemini_connection_for_channels
 
   validates :recipient_id, uniqueness: true, allow_blank: true
 
   attr_encrypted :access_token, :refresh_token, key: proc { |record| record.class.encryption_key }
 
-  after_save :update_default_currency, if: -> { saved_change_to_default_currency? }
-
-  validates :default_currency, inclusion: { in: SUPPORTED_CURRENCIES }, allow_nil: true
+  scope :payable, -> {
+    where(is_verified: true).
+      where(status: 'Active')
```

Comment:
Is there a case in which this goes from an Active state to non-Active?

Reply:
Yes, in the `sync_connection` code, Gemini can return to us a different status.